pom.xml

```diff
@@ -102,15 +106,19 @@
             <artifactId>lombok</artifactId>
             <optional>true</optional>
         </dependency>
+        <dependency>
+            <groupId>mysql</groupId>
+            <artifactId>mysql-connector-java</artifactId>
+            <version>8.0.33</version>
```

Comment:
If I'm not mistaken, Spring dependency management provides fully compatible version of JDBC drivers